package.json

```diff
@@ -12,38 +12,43 @@
   "author": "sebworks",
   "private": true,
   "scripts": {
-    "dev": "pnpm install && pnpm i18n:create-locales-list && pnpm i18n:update-locales && nuxt --hostname 0.0.0.0",
+    "prepare": "husky install",
+    "ver": "func() { pnpm ls --depth=0 | grep -e \"^${1}\\s\" | awk '{print $2}'; }; func",
```

Comment:
```suggestion
    "version": "func() { pnpm ls --depth=0 | grep -e \"^${1}\\s\" | awk '{print $2}'; }; func",
```

I think I'd prefer `version` personally, it isn't that much longer.

Reply:
For some reason, when I run `pnpm version` it runs `npm version` automatically. It might be some incorrect setup on my end or a feature/bug of `pnpm` but given that behaviour, `ver` seemed like the safer/non-conflicting choice. 

<img width="867" alt="image" src="https://user-images.githubusercontent.com/16580576/160981923-bb5a4dfb-a44c-46b8-b3c0-03322ed7045e.png">

Reply:
Oh, good call. Apologies for not considering that!